.github/workflows/pr-check.yaml

```diff
@@ -146,46 +159,93 @@ jobs:
+      - name: Synchronize CRC extension dependencies 
+        working-directory: ./crc-extension
+        run: yarn install --check-files
 
       - name: Execute yarn in OpenShift Local Extension
         working-directory: ./crc-extension
-        run: yarn
+        run: yarn --frozen-lockfile
 
       - name: Revert unprivileged user namespace restrictions in Ubuntu 24.04
+        if: matrix.os == 'ubuntu-24.04'
         run: |
           # allow unprivileged user namespace
           sudo sysctl -w kernel.apparmor_restrict_unprivileged_userns=0
 
-      - name: Build OpenShift Local extension from container file
+      - name: Build OpenShift Local extension from container file and SSO dependency (Ubuntu, podman)
+        if: matrix.os == 'ubuntu-24.04'
         working-directory: ./crc-extension
         run: |
+          # build crc extension
           podman build -t openshift_local_image . -f ./oci/Containerfile.multistage
           CONTAINER_ID_CRC=$(podman create localhost/openshift_local_image --entrypoint "")
           podman export $CONTAINER_ID_CRC > /tmp/openshift_local_extension.tar
-          mkdir -p tests/playwright/output/crc-tests-pd/plugins
+          
+          # build sso extension
           podman pull ghcr.io/redhat-developer/podman-desktop-redhat-account-ext:latest
           CONTAINER_ID_SSO=$(podman create ghcr.io/redhat-developer/podman-desktop-redhat-account-ext --entrypoint "")
           podman export $CONTAINER_ID_SSO > /tmp/sso_extension.tar
+
+          # move extension files to expected locations
+          mkdir -p tests/playwright/output/crc-tests-pd/plugins
           tar -xf /tmp/openshift_local_extension.tar -C tests/playwright/output/crc-tests-pd/plugins/
           mv tests/playwright/output/crc-tests-pd/plugins/extension/ tests/playwright/output/crc-tests-pd/plugins/crcextension
           tar -xf /tmp/sso_extension.tar -C tests/playwright/output/crc-tests-pd/plugins/
           mv tests/playwright/output/crc-tests-pd/plugins/extension/ tests/playwright/output/crc-tests-pd/plugins/ssoextension
           
+      - name: Build OpenShift Local extension locally (Windows, docker) #from container file is not available yet
+        if: matrix.os == 'windows-2022'
+        working-directory: ./crc-extension
+        shell: pwsh
+        run: |
+            # generate 'openshift-local.cdix':
+            yarn build
+            # put the contents into its expected folder:
+            New-Item -ItemType Directory -Path tests\playwright\output\crc-tests-pd\plugins -Force
+            Expand-Archive -Path "$(pwd)\openshift-local.cdix" -DestinationPath ".\tests\playwright\output\crc-tests-pd\plugins\crcextension"
+
+      - name: Build SSO extension (crc dependency) locally (Windows, docker) #from container file is not available yet
+        if: matrix.os == 'windows-2022'
+        working-directory: ./sso-extension
+        shell: pwsh
+        run: |
+            # generate 'redhat-authentication.cdix':
```

Comment:
Same as above, we should follow copy-paste as defined in the container file, right?